applications/pusher/src/modules/pm_apple.erl

```diff
@@ -98,3 +110,19 @@ maybe_load_apns(App, ETS, CertBin) ->
             lager:error("Error loading apns ~p", [Reason]),
             'undefined'
     end.
+
+-spec apns_topic(kz_json:object()) -> binary().
+apns_topic(JObj) ->
+    TokenApp = kz_json:get_ne_binary_value(<<"Token-App">>, JObj),
+    TokenType = kz_json:get_ne_binary_value(<<"Token-Type">>, JObj),
+    case kapps_config:get_ne_binary(<<"pusher">>
+                                   ,[TokenType, <<"apns_topic">>]
+                                   ,'undefined'
+                                   ,TokenApp
+                                   )
+    of
+        'undefined' ->
+            %% Retain the old behaviour
```

Comment:
@danielfinke i think this comment is making the `formatting`  break in circle ci. can you move to it to description of apns_topic (above the spec) ? sorry about that.